test/allCommands.nut.ts

```diff
@@ -120,12 +120,44 @@ describe('verifies all commands run successfully ', () => {
     expect(output).to.have.property('result').includes.keys(['username', 'password']);
   });
 
+  it('generates new password for secondary user (onbehalfof) with length 12', () => {
+    const output = execCmd<{ username: string; password: string }>(
+      'force:user:password:generate -o Other --json -l 12',
+      { ensureExitCode: 0 }
+    ).jsonOutput.result;
+
+    expect(output.password.length).to.equal(12);
+    const complexity5Regex = new RegExp('^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#$|%^&*()[]_-])(?=.{12})');
+    // testing the default complexity
+    expect(complexity5Regex.test(output.password));
+  });
+
+  it('generates new password for secondary user (onbehalfof) with complexity 3', () => {
+    const output = execCmd<{ username: string; password: string }>(
+      'force:user:password:generate -o Other --json -c 3',
+      { ensureExitCode: 0 }
+    ).jsonOutput.result;
+    // testing default length
+    expect(output.password.length).to.equal(13);
+    const complexity3Regex = new RegExp('^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.{14})');
```

Comment:
isn't the `?=.{14}` part of the Regex testing that it's length 14? but the test is testing for 13

Reply:
Its weird how it passed for 14. 
I have removed that length from regex and testing normally